packages/channels/botpress-channel-telegram/src/index.js

```diff
@@ -42,7 +42,9 @@ const outgoingMiddleware = (event, next) => {
 
 module.exports = {
   config: {
-    botToken: { type: 'string', required: true, default: '', env: 'TELEGRAM_TOKEN' }
+    botToken: { type: 'string', required: true, default: '', env: 'TELEGRAM_TOKEN' },
+    proxy: { type: 'any', required: false, default: '', env: 'TELEGRAM_PROXY' },
+    advancedOptions: { type: 'any', required: false, default: {}, env: 'TELEGRAM_ADVANCED_OPTIONS' }
```

Comment:
Hey! Thanks for creating a PR!
Could you describe config-variables in the README?

Reply:
Hi! I was fix README for telegram-channel.